syntax = "proto3";

message MuGpioStore {
  repeated bool state = 1;
}
